install/kubernetes/cilium/values.yaml

```diff
@@ -1,612 +1,1270 @@
-    # for existing GKE clusters that want to disable masquerade
-    disableDefaultSnat: false
+# TODO: Add documentation
+ipv4:
+  enabled: true
 
-  azure:
-    enabled: false
-    # resourceGroup: group1
-    # subscriptionID: 00000000-0000-0000-0000-000000000000
-    # tenantID: 00000000-0000-0000-0000-000000000000
-    # clientID: 00000000-0000-0000-0000-000000000000
-    # clientSecret: 00000000-0000-0000-0000-000000000000
-
-  # cleanState instructs the cilium-agent DaemonSet to clean all state in the
-  # initContainer
-  #
-  # WARNING: Use with care!
-  cleanState: false
+# TODO: Add documentation
+ipv6:
+  enabled: false
 
-  # cleanBpfState instructs the cilium-agent DaemonSet to clean all BPF
-  # datapath state in the initContainer
-  #
-  # WARNING: Use with care!
-  cleanBpfState: false
+# ipvlan is the IPVLAN configuration
+ipvlan:
+  # enabled enables use of the IPVLAN datapath
+  enabled: false
 
-  nodeinit:
-    # enabled enables inclusion of the nodeinit DaemonSet
-    enabled: false
+  # masterDevice is the name of the device to use to attach secondary IPVLAN
+  # devices
+  # masterDevice: eth0
+
+# k8s is the Kubernetes specific configuration
+k8s: {}
+  # requireIPv4PodCIDR enables waiting for Kubernetes to provide the PodCIDR
+  # range via the Kubernetes node resource
+  # requireIPv4PodCIDR: false
+
+  # requireIPv6PodCIDR enables waiting for Kubernetes to provide the PodCIDR
+  # range via the Kubernetes node resource
+  # requireIPv46PodCIDR: false
+
+# Keep the deprecated selector labels when deploying Cilium DaemonSet
+keepDeprecatedLabels: false
+
+# Keep the deprecated probes when deploying Cilium DaemonSet
+keepDeprecatedProbes: false
+
+# kubeProxyReplacement enables kube-proxy replacement in Cilium BPF datapath
+kubeProxyReplacement: "probe"
+
+kvstore:
+  # Managed enables/disables the Cilium etcd-operator
+  managed: false
+  # ssl configures tls for kvstore client connections
+  ssl:
+
+l7Proxy: false
+
+# To include or exclude matched resources from cilium identity evaluation
+# labels: ""
+
+# logOptions allows you to define logging options. eg:
+# logOptions:
+#   format: json
+
+# logSytemLoad enables logging of system load
+logSystemLoad: false
+
+
+# maglev is the configuration for consistent hashing
+maglev: {}
+  # tableSize is the size (parameter M) for the backend table of one service entry
+  # tableSize:
+
+  # hashSeed is the cluster-wide base64 encoded seed for the hashing
+  # hashSeed:
 
-    # bootstrapFile is the location of the file where the bootstrap timestamp is
-    # written by the node-init DaemonSet
-    bootstrapFile: "/tmp/cilium-bootstrap-time"
+# masquerade enables masquerading of traffic leaving the node for
+# destinations outside of the cluster.
+masquerade: true
 
-  daemon:
-    runPath: "/var/run/cilium"
+# nativeRoutingCIDR allows to explicitly specify the CIDR for native routing. This
+# value corresponds to the configured cluster-cidr.
+# nativeRoutingCIDR:
```

Comment:
@seanmwinn Here "nativeRoutingCIDR" is commented out. Was that an intended change?